test/unified_memory_allocation/memoryPool.hpp

```diff
@@ -6,10 +6,13 @@
 #include <cstring>
 #include <functional>
 #include <random>
+#include <thread>
 
 #ifndef UMA_TEST_MEMORY_POOL_OPS_HPP
 #define UMA_TEST_MEMORY_POOL_OPS_HPP
 
+#define NTHREADS 5
```

Comment:
This should be a static constexpr defined in the umaPoolTest class.

Reply:
Done.